src/olympia/discovery/serializers.py

```diff
@@ -48,4 +49,10 @@ def to_representation(self, instance):
                 '{start_sub_heading}', '<span>').replace(
                 '{end_sub_heading}', '</span>').replace(
                 '{addon_name}', addon_link)
+
+        if (data['description'] is None and
+                instance.addon.type == amo.ADDON_EXTENSION and
+                instance.addon.summary):
```

Comment:
This can clearly be made easier to read with splitting it up again

```python
autofill_description = (
    data['description'] is None and
    instance.addon.type == amo.ADDON_EXTENSION and
    instance.addon.summary)

if autofill_description:
    data['description'] = ...
```